core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionParser.scala

```diff
@@ -12,11 +12,19 @@ import cats.instances.try_._
 
 object ExpressionParser {
   def parseMCExpressions(inputs: Map[String, AttributeString], outputs: Map[String, AttributeString], allowRootEntity: Boolean, parser: SlickExpressionParser): ParsedMCExpressions = {
+    val noEntityAllowedErrorMsg = "Only allowed when running with workspace data model. However, workspace attributes can be used."
+    val invalidErrorMsg = "Invalid expression. Use a string or workspace attribute."
```

Comment:
Another nitpicky thing: the inputs can be more than strings and workspace attributes - they can also ints, maps, arrays, etc.